forest/src/daemon.rs

```diff
@@ -283,8 +283,9 @@ pub(super) async fn start(config: Config) {
     );
 
     // Initialize Consensus. Mining may or may not happen, depending on type.
-    let (consensus, mining_task) =
-        cns::consensus(&state_manager, &keystore, &mpool, submitter).await;
+    let (consensus, mining_tasks) = cns::consensus(&state_manager, &keystore, &mpool, submitter)
+        .await
+        .unwrap();
```

Comment:
We should eventually add more consistent error handling to the daemon.

Reply:
Yeah, I was just following the pattern; but at least it all culminates in this file, so it's easy to turn it into a nice error message instead.